pydgin/elf.py

```diff
@@ -528,28 +528,23 @@ def elf_reader( file_obj, is_64bit=False ):
     #section_name = start.partition('\0')[0]
     section_name = start.split('\0', 1)[0]
 
-    # only sections marked as lloc should be written to memory
-
-    if not (shdr.flags & ElfSectionHeader.FLAGS_ALLOC):
-      continue
-
     # Read the section data if it exists
-
-    if section_name not in ['.sbss', '.bss']:
+    if (shdr.type == ElfSectionHeader.TYPE_NULL or
+        section_name == '.comment' or
```

Comment:
Is it necessary to check against section names? I believe these sections have their ``ALLOC`` flag unset, and it would be better to use that to prevent them from loading. You can check the ``ALLOC`` flags of the sections using ``readelf -a``.